src/components/include/media_manager/media_manager.h

```diff
@@ -36,16 +36,62 @@
 #include <string>
 #include "protocol/service_type.h"
 #include "media_manager/media_manager_settings.h"
+#include "utils/macro.h"  // for "DEPRECATED"
 namespace media_manager {
 
+/**
+ * \brief Describes different sampling options for PerformAudioPassThru.
+ */
+typedef enum {
```

Comment:
Are separate enum definitions required for SamplingRate, AudioCaptureQuality and AudioType? Could we not use the existing mobile_apis::SamplingRate::eType, mobile_apis:: BitsPerSample::eType and mobile_apis:: AudioType::eType?

Reply:
Thanks for your suggestion. Please refer to my comment https://github.com/smartdevicelink/sdl_core/pull/2184#issuecomment-644639701

Reply:
I have updated MediaManager to use Mobile API enums. Please refer to my commit 8211590. Thanks!